requirements-test.in

```diff
@@ -22,7 +22,7 @@ transformers_stream_generator # required for qwen-vl test
 matplotlib # required for qwen-vl test
 mistral_common[opencv] >= 1.5.0 # required for pixtral test
 datamodel_code_generator # required for minicpm3 test
-lm-eval[api]==0.4.4 # required for model evaluation test
+lm-eval[api] @ git+https://github.com/robertgshaw2-neuralmagic/lm-evaluation-harness.git@streaming-api # required for model evaluation test with streaming
```

Comment:
are you sending a PR up?

Reply:
Im not going to send a PR up to `lm-eval`